lib/js/_create/_build-new-package.js

```diff
@@ -71,6 +80,15 @@ function generateFiles(config, answers) {
 					}
 				})
 			});
+		} else {
+			fileTasks.push({
+				title: `Create file ${chalk.bold.white(value)}`,
+				task: () => fs.writeFile(`${filePath}/${answers.pkgname}/${value}`, '__tests__ coverage **/*.html HISTORY.md', err => {
+					if (err) {
+						console.error(err);
+					}
+				})
+			});
 		}
```

Comment:
I assume this is for creating the `.npmignore` file. I don't think we should be doing it like this